docs/outlook/online-meeting.md

```diff
@@ -0,0 +1,182 @@
+                getBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+This sample shows an implementation of the supporting function `updateBody` used in the previous example that appends the online meeting details to the current body of the meeting.
+
+```js
+function updateBody(event, existingBody) {
+    // Append new body to the existing body.
+    mailboxItem.body.setAsync(existingBody + newBody,
+        { asyncContext: event, coercionType: "html" },
+        function (setBodyResult) {
+            if (setBodyResult.status === Office.AsyncResultStatus.Succeeded) {
+                setBodyResult.asyncContext.completed({ allowEvent: true });
+            } else {
+                console.error("Failed to set HTML body.");
+                setBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+### Testing and validation
+
+You can follow the usual guidance to [test and validate your add-in](testing-and-tips.md). After [sideloading](sideload-outlook-add-ins-for-testing.md) in Outlook on the web, Windows, or Mac, you should restart Outlook on your mobile device (Android is the only supported client for now) then verify on a new meeting screen that the Microsoft Teams or Skype toggle has been replaced with your own.
+
+#### Create meeting UI
+
+As a meeting organizer, you should see screens similar to the following when you create a meeting.
+
+[![screenshot of create meeting screen on Android - loading Contoso toggle](../images/outlook-android-create-online-meeting-load.png)](../images/outlook-android-create-online-meeting-load-expanded.png#lightbox) [![screenshot of create meeting screen on Android - Contoso toggle off](../images/outlook-android-create-online-meeting-off.png)](../images/outlook-android-create-online-meeting-off-expanded.png#lightbox) [![screenshot of create meeting screen on Android - Contoso toggle on](../images/outlook-android-create-online-meeting-on.png)](../images/outlook-android-create-online-meeting-on-expanded.png#lightbox)
+
+#### Join meeting UI
+
+As a meeting attendee, you should see a screen similar to the following when you view the meeting.
+
+[![screenshot of join meeting screen on Android](../images/outlook-android-join-online-meeting-view.png)](../images/outlook-android-join-online-meeting-view-expanded.png#lightbox)
+
+## Available APIs
+
+***TODO:*** Confirm set of APIs - will need to update ref docs for each of them - point to this article
+e.g. **Note**: This member is not supported in Outlook on iOS or Android. However, there's an exception for online-meeting providers. See *online meeting article* for details.
+
+***TODO:*** Note similar exception in [mobile add-ins support](add-mobile-support.md#compose-mode-and-appointments) article.
+
+The following APIs are available for this feature.
+
+- [Dialog APIs for auth flow](../develop/dialog-api-in-office-add-ins.md)
+- Appointment Organizer APIs
```

Comment:
For visual and clarity, consider removing "- Appointment Organize APIs" entirely (or make it the list title in bold) and have all APIs at the same level in the bulleted list.

Reply:
@lindalu-MSFT This section looks like the following image. Is this inappropriate?
![image](https://user-images.githubusercontent.com/28541963/79013762-41b77980-7b1e-11ea-9cc4-613f6518f212.png)

Reply:
Yes, the level 1 bullets are not the same, one is a link, the other is a heading. This is confusing to screen reader users.

Reply:
@lindalu-MSFT I've updated this - can you take another look? Thanks.

Reply:
Looks awesom